consensus/safety-rules/benches/safety_rules.rs

```diff
@@ -68,25 +70,38 @@ fn serializer(n: u64) {
     let signer = ValidatorSigner::from_int(0);
     let file_path = NamedTempFile::new().unwrap().into_temp_path().to_path_buf();
     let storage = OnDiskStorage::default_storage(file_path).unwrap();
-    let safety_rules_manager = SafetyRulesManager::new_local(storage, signer.clone());
+    let safety_rules_manager = SafetyRulesManager::new_serializer(storage, signer.clone());
     lsr(safety_rules_manager.client(), signer, n);
 }
 
 fn thread(n: u64) {
     let signer = ValidatorSigner::from_int(0);
     let file_path = NamedTempFile::new().unwrap().into_temp_path().to_path_buf();
     let storage = OnDiskStorage::default_storage(file_path).unwrap();
-    let safety_rules_manager = SafetyRulesManager::new_local(storage, signer.clone());
+    let safety_rules_manager = SafetyRulesManager::new_thread(storage, signer.clone());
     lsr(safety_rules_manager.client(), signer, n);
 }
 
+fn process(n: u64) {
+    let file_path = NamedTempFile::new().unwrap().into_temp_path().to_path_buf();
+    let mut config = OnDiskStorageConfig::default();
+    config.default = true;
+    config.path = file_path;
+    let backend = SafetyRulesBackend::OnDiskStorage(config);
+    let client_wrapper = ProcessClientWrapper::new(backend);
+    let signer = client_wrapper.signer();
+
+    lsr(Box::new(client_wrapper), signer, n);
```

Comment:
what does `lsr` stand for?

Reply:
Should ask David from last month, I suspect Libra Safety Rules, but that doesn't seem particularly meaningful.